packages/devtools/src/ganache/deployer-with-cache.ts

```diff
@@ -85,10 +100,16 @@ export class GanacheNCacheDeployer {
       } else {
         throw e;
       }
+    } finally {
+      await new Promise((resolve, reject) =>
+        lockfile.unlock(lockPath, result => {
+          !result ? resolve() : reject(result);
```

Comment:
I think it's easier to parse positive booleans.

```suggestion
          result ? reject(result) : resolve();
```

Reply:
Personally I prefer making the success case the first case but I don't mind too much either way.